dist/windows/config.nsh

```diff
@@ -35,12 +35,15 @@
 ;End of user configurable options
 ;==============================================================================
 
+; QBT_CPU_ARCH (x64 or arm64)
+!define /ifndef QBT_CPU_ARCH "x64"
+
 !ifndef QBT_INSTALLER_WINDOWNAME | QBT_INSTALLER_FILENAME
   ; The name of the installer
-  !define QBT_INSTALLER_WINDOWNAME "${QBT_VERSION} x64"
+  !define QBT_INSTALLER_WINDOWNAME "${QBT_VERSION} ${QBT_CPU_ARCH}"
```

Comment:
In any case, I would just drop the arch suffix from window title. It doesn't matter if someone can't run different packages on the same platform, like when we provided both x32 and x64 packages that could be run on an x64 platform. 
Or can the x64 installer be run on ARM?

Reply:
Yes

Reply:
by the way, obs-studio have a emulation to check if it's a emulation environment
https://github.com/obsproject/obs-studio/blob/0b1229632063a13dfd26cf1cd9dd43431d8c68f6/libobs/util/platform-windows.c#L963-L992


i think it's better to add in https://github.com/qbittorrent/qBittorrent/blob/master/src/app/main.cpp what do you think?

Reply:
I apologize for the misunderstanding. I will keep showing architecture in installer title bar, but shoule we add an arm64 emulation check in `main.cpp` like obs-studio?